tests/output/inline-help/iam.txt

```diff
@@ -1323,6 +1325,45 @@ Options:
                                   value will be used
   -?, -h, --help                  Show this message and exit.
 
+++++++++++++++++++++++++++++++++++++++++++++++
+$ oci iam identity-provider-group --help
+Usage: oci iam identity-provider-group [OPTIONS] COMMAND [ARGS]...
+
+  A group created in an identity provider that can be mapped to a group in OCI
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  list  Lists the identity provider groups.
+
+++++++++++++++++++++++++++++++++++++++++++++++
+$ oci iam identity-provider-group list --help
```

Comment:
This command seems to be missing in changelog

Reply:
Good catch thanks! Added to changelog